### DISCUSSION TITLE
HyFeatures - part of WaterML and National Map Effort

### DISCUSSION THREAD

**Discussion by joewheaton** [Riverscapes/riverscapes-tools#612] [VBET]
David Tarboton,  just pointed out this effort: https://docs.opengeospatial.org/is/14-111r6/14-111r6.html during @shelbysawyer  committee meeting. He is working with them and although it is not officially adopted it is what is driving the new update to the National Map. He suggested we get our riverscapes and vbet stuff in there...

@philipbaileynar, @lauren-herbine, @shelbysawyer, @KellyMWhitehead, @MattReimer, @jtgilbert and @wally-mac will all be interested in this.

**Comment by MattReimer** [comment]
Interesting. Sort of a standardization of hydrological features. I like the class diagrams for object hierarchies.
